flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/main/java/com/ververica/cdc/connectors/mysql/source/MySqlMetadataAccessor.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.mysql.source;
+
+import com.ververica.cdc.common.annotation.Internal;
+import com.ververica.cdc.common.event.TableId;
+import com.ververica.cdc.common.schema.Schema;
+import com.ververica.cdc.common.source.MetadataAccessor;
+import com.ververica.cdc.connectors.mysql.source.config.MySqlSourceConfig;
+import com.ververica.cdc.connectors.mysql.utils.MySqlSchemaUtils;
+import io.debezium.connector.mysql.MySqlPartition;
+
+import javax.annotation.Nullable;
+
+import java.util.List;
+
+/** {@link MetadataAccessor} for {@link MySqlDataSource}. */
+@Internal
+public class MySqlMetadataAccessor implements MetadataAccessor {
+
+    private final MySqlSourceConfig sourceConfig;
+
+    private final MySqlPartition partition;
+
+    public MySqlMetadataAccessor(MySqlSourceConfig sourceConfig) {
+        this.sourceConfig = sourceConfig;
+        this.partition =
+                new MySqlPartition(sourceConfig.getMySqlConnectorConfig().getLogicalName());
+    }
+
+    @Override
+    public List<String> listNamespaces() {
+        throw new UnsupportedOperationException("List namespace is not supported by MySQL.");
+    }
+
+    @Override
+    public List<String> listSchemas(@Nullable String namespace) {
+        return MySqlSchemaUtils.listDatabases(sourceConfig);
+    }
+
+    @Override
+    public List<TableId> listTables(@Nullable String namespace, @Nullable String dbName) {
+        return MySqlSchemaUtils.listTables(sourceConfig, dbName);
+    }
+
```

Comment:
could you add some java doc for the three methods? the note could contains the table path mapping relation